src/BaseCurve.sol

```diff
@@ -91,21 +104,14 @@ abstract contract BaseCurve {
     /// @param totalAssets Total quantity of assets already staked into the curve
     /// @return assets The number of assets equivalent to the given shares
     function convertToAssets(uint256 shares, uint256 totalShares, uint256 totalAssets)
-        public
```

Comment:
3.20 - Functions in Curve Contracts Could Be external Instead of public

Made this `external`